Formula/duckscript.rb

```diff
@@ -19,13 +19,11 @@ class Duckscript < Formula
 
   on_linux do
     depends_on "pkg-config" => :build
-    depends_on "openssl@1.1" # Uses Secure Transport on macOS
+    depends_on "openssl@3" # Uses Secure Transport on macOS
   end
 
   def install
-    cd "duckscript_cli" do
-      system "cargo", "install", *std_cargo_args
-    end
+    system "cargo", "install", "--features", "tls-native", *std_cargo_args(path: "duckscript_cli")
```

Comment:
Still need to make sure of linkage. Open to opinions on whether we should favor the statically linked pure Rust implementation vs dynamically linked system/Homebrew library.

Upstream changed from native OpenSSL to rustls, and then added option afterward https://github.com/sagiegurari/duckscript/blob/master/CHANGELOG.md#v0813-2022-07-21
```
### v0.8.13 (2022-07-21)

* Enhancement: Runtime - Enable to clone duckscript context #253 (thanks waterlens)
* Enhancement: Support both native TLS via openssl and pure rust TLS #258 (thanks @jirutka)

### v0.8.12 (2022-05-25)

* Enhancement: Add support for stdin input passing to child process in exec, watchdog and spawn commands #247
* Enhancement: Replace native TLS support via openssl with pure rust TLS
* Update dependencies
```

Reply:
Old linkage #112420:
```
==> brew linkage --cached duckscript
System libraries:
  /lib/x86_64-linux-gnu/libc.so.6
  /lib/x86_64-linux-gnu/libgcc_s.so.1
  /lib/x86_64-linux-gnu/libm.so.6
```

New linkage:
```
==> brew linkage --cached duckscript
System libraries:
  /lib/x86_64-linux-gnu/libc.so.6
  /lib/x86_64-linux-gnu/libgcc_s.so.1
  /lib/x86_64-linux-gnu/libm.so.6
Homebrew libraries:
  /home/linuxbrew/.linuxbrew/opt/openssl@3/lib/libcrypto.so.3 (openssl@3)
  /home/linuxbrew/.linuxbrew/opt/openssl@3/lib/libssl.so.3 (openssl@3)
```